<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RibbonAnalysis</class>
 <widget class="QWidget" name="RibbonAnalysis">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>86</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_7">
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <property name="horizontalSpacing">
    <number>0</number>
   </property>
   <item row="0" column="7">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>61</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <property name="verticalSpacing">
       <number>3</number>
      </property>
      <item row="0" column="0">
       <widget class="QWidget" name="widget_9" native="true">
        <layout class="QGridLayout" name="gridLayout_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="1" column="0">
          <widget class="RibbonButton" name="Descriptives">
           <property name="minimumSize">
            <size>
             <width>76</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Descriptives</string>
           </property>
           <property name="icon">
            <iconset resource="../resources/icons.qrc">
             <normaloff>:/icons/analysis-descriptives.svg</normaloff>:/icons/analysis-descriptives.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>37</width>
             <height>28</height>
            </size>
           </property>
           <property name="popupMode">
            <enum>QToolButton::InstantPopup</enum>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QWidget" name="otherPanel" native="true">
     <layout class="QGridLayout" name="gridLayout_5">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <property name="verticalSpacing">
       <number>3</number>
      </property>
      <item row="0" column="0">
       <widget class="QWidget" name="widget_11" native="true">
        <layout class="QGridLayout" name="gridLayout_6">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="horizontalSpacing">
          <number>5</number>
         </property>
         <item row="0" column="1">
          <widget class="RibbonButton" name="factoranalysisButton">
           <property name="minimumSize">
            <size>
             <width>76</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Factor</string>
           </property>
           <property name="icon">
            <iconset resource="../resources/icons.qrc">
             <normaloff>:/icons/analysis-classical-sem.svg</normaloff>:/icons/analysis-classical-sem.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>37</width>
             <height>28</height>
            </size>
           </property>
           <property name="popupMode">
            <enum>QToolButton::InstantPopup</enum>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="RibbonButton" name="BFFromT">
           <property name="minimumSize">
            <size>
             <width>76</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>BF from t</string>
           </property>
           <property name="icon">
            <iconset resource="../resources/icons.qrc">
             <normaloff>:/icons/analysis-bayesian-ttest.svg</normaloff>:/icons/analysis-bayesian-ttest.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>37</width>
             <height>28</height>
            </size>
           </property>
           <property name="popupMode">
            <enum>QToolButton::InstantPopup</enum>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QWidget" name="sep1" native="true">
     <layout class="QGridLayout" name="gridLayout_8">
      <item row="0" column="0">
       <widget class="Line" name="line1">
        <property name="styleSheet">
         <string notr="true">QWidget {	color : #929292 ; }</string>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QWidget" name="sep2" native="true">
     <property name="styleSheet">
      <string notr="true">QWidget {	color : #929292 ; }</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_9">
      <item row="0" column="0">
       <widget class="Line" name="line2">
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QWidget" name="mainPanel" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <property name="verticalSpacing">
       <number>3</number>
      </property>
      <item row="0" column="0">
       <widget class="QWidget" name="widget_10" native="true">
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <property name="horizontalSpacing">
          <number>5</number>
         </property>
         <item row="0" column="0">
          <widget class="RibbonButton" name="ttestButton">
           <property name="minimumSize">
            <size>
             <width>76</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>T-Tests</string>
           </property>
           <property name="icon">
            <iconset resource="../resources/icons.qrc">
             <normaloff>:/icons/analysis-classical-ttest.svg</normaloff>:/icons/analysis-classical-ttest.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>37</width>
             <height>28</height>
            </size>
           </property>
           <property name="popupMode">
            <enum>QToolButton::InstantPopup</enum>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="RibbonButton" name="anovaButton">
           <property name="minimumSize">
            <size>
             <width>76</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>ANOVA</string>
           </property>
           <property name="icon">
            <iconset resource="../resources/icons.qrc">
             <normaloff>:/icons/analysis-classical-anova.svg</normaloff>:/icons/analysis-classical-anova.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>37</width>
             <height>28</height>
            </size>
           </property>
           <property name="popupMode">
            <enum>QToolButton::InstantPopup</enum>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="RibbonButton" name="regressionButton">
           <property name="minimumSize">
            <size>
             <width>76</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Regression</string>
           </property>
           <property name="icon">
            <iconset resource="../resources/icons.qrc">
             <normaloff>:/icons/analysis-classical-regression.svg</normaloff>:/icons/analysis-classical-regression.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>37</width>
             <height>28</height>
            </size>
           </property>
           <property name="popupMode">
            <enum>QToolButton::InstantPopup</enum>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="RibbonButton" name="frequenciesButton">
           <property name="minimumSize">
            <size>
             <width>76</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Frequencies</string>
           </property>
           <property name="icon">
            <iconset resource="../resources/icons.qrc">
             <normaloff>:/icons/analysis-classical-crosstabs.svg</normaloff>:/icons/analysis-classical-crosstabs.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>37</width>
             <height>28</height>
            </size>
           </property>
           <property name="popupMode">
            <enum>QToolButton::InstantPopup</enum>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextUnderIcon</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <zorder>widget</zorder>
  <zorder>mainPanel</zorder>
  <zorder>otherPanel</zorder>
  <zorder>sep1</zorder>
  <zorder>sep2</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>RibbonButton</class>
   <extends>QToolButton</extends>
   <header>widgets/ribbonbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/icons.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>itemSelected()</slot>
  <slot>menuItemSelected()</slot>
 </slots>
</ui>
